File: DynamoDB-SDK-Examples/node.js/WorkingWithTables/table_change_to_on_demand.js
// A simple script to switch a table to on-demand capacity mode.

const { DynamoDBClient, UpdateTableCommand } = require('@aws-sdk/client-dynamodb');

const REGION = "us-west-2"; // Replace with your target region
const TableName = "Music"; // Replace with your table name.

const dbclient = new DynamoDBClient({ region: REGION });

async function updateTable() {
    const params = {
        BillingMode: "PAY_PER_REQUEST",
        TableName: TableName,
    };
    return await dbclient.send( new UpdateTableCommand(params));
    // Add code to wait for table update to complete before returning.
}

updateTable()
    .then((data) => console.log(data))
    .catch((error) => console.log("An error occured while updating the table:" + ' ' + error.message ));

File: examples/SDK/python/control_plane/WorkingWithTables/table_change_to_on-demand.py
from __future__ import print_function # Python 2/3 compatibility
import boto3, pprint

dynamodb = boto3.resource('dynamodb', region_name='us-west-2')

table = dynamodb.Table("Music") # Substitute your table name

table.update(
    BillingMode="PAY_PER_REQUEST"
)